spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -71,7 +73,7 @@
         let(:tmpl_at_openstack) { create(:virtual_machine_template, tenants: [openstack], appliance_type: appl_type) }
 
         it 'selects cheapest flavour that satisfies requirements' do
-          selected_tmpl, flavor = subject.select_tmpl_and_flavor([tmpl_at_amazon, tmpl_at_openstack])
+          selected_tmpl, tenant, flavor = subject.select_tmpl_and_flavor_and_tenant([tmpl_at_amazon, tmpl_at_openstack])
```

Comment:
Useless assignment to variable - `selected_tmpl`. Use `_` or `_selected_tmpl` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `tenant`. Use `_` or `_tenant` as a variable name to indicate that it won't be used.<br>Line is too long. [120/80]